cpu/kinetis_common/periph/uart.c

```diff
@@ -23,13 +23,27 @@
  * @}
```

Comment:
as stated above, I think the support for the LPUART should be PRed separately...